draft-ietf-quic-transport.md

```diff
@@ -128,12 +128,12 @@ QUIC establishes a connection, which is a stateful interaction between a client
 and server. The primary purpose of a connection is to support the structured
 exchange of data by an application protocol.
 
-Streams are means by which an application protocol exchanges information.
-Streams are ordered sequences of bytes. Two types of stream can be created:
-bidirectional streams, which allow both endpoints to send data; and
-unidirectional streams, which allow a single endpoint to send. A credit-based
-scheme is used to limit stream creation and to bound the amount of data that
-can be sent.
+Within a connection, streams are the means by which an application protocol
```

Comment:
I don't like this change much.

Reply:
The paragraph above talks about connection, and the following paragraph talks about setting one up. Streams are brought up here first but their relationship to connections is not clear, even after reading this paragraph. I wanted some glue.